tests/build/build_templates.go

```diff
@@ -369,7 +369,15 @@ var _ = framework.BuildSuiteDescribe("Build templates E2E test", Label("build",
 				}
 
 				if !strings.Contains(scenario.GitURL, "from-scratch") {
-					Expect(sbom.GetPackages()).ToNot(BeEmpty())
+					packages := sbom.GetPackages()
+					Expect(packages).ToNot(BeEmpty())
+					for i := range packages {
+						Expect(packages[i].GetName()).ToNot(BeEmpty())
+						Expect(packages[i].GetVersion()).ToNot(BeEmpty())
```

Comment:
I think versions _can_ be empty, for example for Go standard library packages (hermeto doesn't report a version for those, and shouldn't)

Checking the name might be enough

Reply:
Removed the version check now.